File: atm.py
#   simulates an ATM

#   let's get the balance
name = input("Please enter your name to begin: ").strip().title()
balance = 1500

print(f"Hello, {name}! Welcome back to Python Bank.")
print(f"Your current balance is {balance:,}.")

#   now, let's make a transaction
confirmation = input("If you would like to withdraw more, type 'withdraw'. If you would like to instead deposit, type 'deposit'. ")
#   if confirmation equals yes
if confirmation == "withdraw":
    withdrawal = float(input("Please enter the amount you would like to withdraw: "))
    balance2 = balance - withdrawal
    print(f"Your new balance is: {balance2:,}.")
    #   making another transaction
if confirmation == "deposit":
    deposit = float(input("Please enter the amount you would like to deposit in: "))
    balance2 = balance + deposit
    print(f"Your new balance is {balance2:,}.")


receipt_confirmation = input("Would you like a receipt? Please type 'yes' or 'no'. ")
if receipt_confirmation == "yes":
    print(f"***Receipt for {name}.")
    print(f"Previous balance: {balance}.")
    # figure out how to show "Today's transactions", showing either the withdrawal amount or the deposit amount
    print(f"Your new balance is: {balance2}.")
    print("\nThank you for banking with Python Bank! See you again soon on the Terminal.")
if receipt_confirmation == "no":
    print("Thank you for banking with Python Bank! See you again soon on the Terminal.")

#   No need to use quotation marks and plus signs, it's a f string
#   {withdrawal} is a like a template for the value

File: atm.cpp
#include <iostream>

// # TODO : make this accept only numbers

int main()
{
  std::string name;
  int balance = 1000;

  std::cout << "Please enter your name to begin: ";
  std::cin >> name;
  std::cout << "Hello " << name << ", welcome to C++ bank" << std::endl;
  std::cout << "Your current balance is " << balance << std::endl;

  std::string response;
  std::cout << "To withdraw, type 'withdraw'. If you would like to deposit, type 'deposit'." << std::endl;
  std::cin >> response;
  if (response == "withdraw" || response == "Withdraw")
  {
    int withdrawn;
    std::cout << "How much would you like to withdraw?" << std::endl;
    std::cin >> withdrawn;
    int withTotal = balance - withdrawn;
    std::cout << "Your new balance is " << withTotal << std::endl;
  }
  else if (response == "deposit" || response == "Deposit")
  {
    int deposited;
    std::cout << "How much would you like to deposit?" << std::endl;
    std::cin >> deposited;
    int depTotal = balance + deposited;
    std::cout << "Your balance is " << depTotal << std::endl;
  }
  else
  {
    std::cout << "You did not provide a response! Run the ATM once more and try again.";
  }
}
